src/platforms/unity/common/unity-lite/index.mdx

```diff
@@ -0,0 +1,43 @@
+---
+title: Unity Lite
```

Comment:
If you're using the common getting started page, you need only
1.  populate the code samples (the ones noted in the PlatformContent `includePath`)
2. create the config.yml file 

You don't need an index file.